cfgov/jinja2/v1/_includes/organisms/header.html

```diff
@@ -8,17 +8,21 @@
 
    Creates markup for Header organism.
 
-   show_banner: Whether the global banner molecule is included.
-                Default is true.
+   show_banner:     Whether the global banner molecule is included.
+                    Default is false.
 
-   language:    The page's language value (used to display different
-                information on Spanish pages). Defaults to English.
+   show_mega_menu:  Whether the mega menu is included. Default is true.
+
+   language:        The page's language value (used to display different
+                    information on Spanish pages). Defaults to English.
```

Comment:
Could say `Defaults to 'en' for English.`

Reply:
Fixed in 9bbb731.